src/lib.rs

```diff
@@ -1483,14 +1461,39 @@ impl Node {
 			},
 		}
 	}
+
+	pub fn set_context(&mut self, context: *mut Context) {
+		unsafe {
+			internal::YGNodeSetContext(self.inner_node, context as *mut std::os::raw::c_void)
+		}
+	}
+
+	pub fn get_context(node: &Node) -> &mut Context {
+		unsafe { &mut *(internal::YGNodeGetContext(node.inner_node) as *mut Context) }
+	}
+
+	pub fn get_context_raw(node_ref: &NodeRef) -> &mut Context {
+		unsafe { &mut *(internal::YGNodeGetContext(*node_ref) as *mut Context) }
+	}
+}
+
+#[derive(Debug)]
+pub struct Context(pub Box<Any>);
```

Comment:
Is it possible to use generics here? I'm not too knowledgeable on C FFI when dealing with the Any type vs. using generics

Reply:
I think so! I'll get back to you in a bit.